client/src/components/App/Content/LandingPage/InvestigationTable/SettingsActions/useSettingsActions.ts

```diff
@@ -0,0 +1,51 @@
+import axios from 'axios';
```

Comment:
export every onClick function outside (declare the function outside the return and use it inside)